proto/minder/v1/minder.proto

```diff
@@ -667,6 +694,17 @@ service EvalResultsService {
             relation: RELATION_PROFILE_STATUS_GET
         };
     }
+
+    rpc ListEvaluationHistory(ListEvaluationHistoryRequest) returns (ListEvaluationHistoryResponse) {
```

Comment:
Is `EvalResultsService` the right service?

Reply:
I think we can reuse it